### PR TITLE
Improved Form Responsiveness

### PR DESCRIPTION
**Pull request by theBeginner86**
## Related Issue 

- In this PR, I have solved the responsiveness of the form of our website. I have used standard breakpoints with media query to solve this issue. 

Closes: #173 

### Describe the changes you've made

I have made changes in the following files:
- styles.css
Here, I have used media query and added following breakpoints: <=767px(for mobile devices), >=768px & <=1199px(for tablet and medium sized displays), and >=1200px(for desktops and larger screen devices).
Then, I have just added few paddings, set the widths and corrected the margins.
- form.jsx
There I changed the class from col-sm-2 to col-sm-3 for two proper alignment of the input boxes.

## Type of change

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Enhancement in the existing features
- [ ] Other

## Mention any unusual behaviour of your code (Write NA if not)
Any unusual behaviour of your code: NA

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly whereever it was hard to understand.
- [x] My changes generate no new warnings.
- [x] I have verified/tested my code by running it locally.

## Additional Info (optional)
Any additional information you want to give:

- For smaller display mobile devices:
Screenshot 2021-05-05 at 4 09 00 PM

- For medium to high display mobile devices:
Screenshot 2021-05-05 at 4 08 30 PM

- For tablets and medium display computers:
Screenshot 2021-05-05 at 4 08 07 PM

-For desktops and larger devices:
Screenshot 2021-05-05 at 4 07 51 PM

### CODE REVIEW

File: src/components/style.css [hunk 1]
```diff
@@ -358,6 +358,53 @@ ul {
   font-size: 14px;
   color: #878787;
 }
+@media screen and (max-width: 767px) {
+  div.formRoot,
+  select.custom-select,
+  input.form-control {
+    font-size: 0.6rem;
```
Comment (Namanl2001) [inline src/components/style.css:365]: 0.6 is too small. Make it 0.8

File: src/components/style.css [hunk 2]
```diff
@@ -358,6 +358,53 @@ ul {
...
+    margin: 300px auto;
+    width: 450px;
+    position: relative;
+  }
+}
+@media screen and (min-width: 768px) and (max-width: 1199px) {
+  div.formRoot,
+  select.custom-select,
+  input.form-control {
+    font-size: 0.7rem;
```
Comment (Namanl2001) [inline src/components/style.css:385]:
```suggestion
    font-size: 0.9rem;
```

**Review by Namanl2001 — CHANGES_REQUESTED** [review]
image


As you can see the right part of the form is out of screen. please fix it. Other things looks good.

File: src/components/style.css (hunk 1 above)
Comment (theBeginner86) [reply src/components/style.css:365]: ok I will do it.

File: src/components/style.css (hunk 2 above)
Comment (theBeginner86) [reply src/components/style.css:385]: ok I will do it.

**Comment by theBeginner86** [comment]
@Namanl2001 
I have made the requested changes, Please review it.

**Comment by Namanl2001** [comment]
Can you please add a screen shot for 360 X 640 resolution?

**Comment by theBeginner86** [comment]
Screenshot 2021-05-05 at 10 34 30 PM

File: src/components/style.css [hunk 3]
```diff
@@ -358,6 +358,53 @@ ul {
...
+  select.custom-select,
+  input.form-control {
+    width: 100%;
+  }
+  div.form-child-left,
+  div.form-child-right {
+    flex: 100%;
+  }
+  div.modal-content {
+    margin: 300px auto;
```
Comment (Namanl2001) [inline src/components/style.css:376]:
```suggestion
    margin: 100px auto;
```

File: src/components/style.css [hunk 4]
```diff
@@ -358,6 +358,53 @@ ul {
   font-size: 14px;
   color: #878787;
 }
+@media screen and (max-width: 767px) {
```
Comment (Namanl2001) [inline src/components/style.css:361]:
```suggestion
@media screen and (max-width: 500px) {
```

**Review by Namanl2001 — CHANGES_REQUESTED** [review]
I just played a lil bit with the css. IMO we should add one more configuration as follows:
```
@media screen and (min-width: 500px) and (max-width: 767px) {
  div.formRoot,
  select.custom-select,
  input.form-control {
    font-size: 0.8rem;
  }
  select.custom-select,
  input.form-control {
    width: 100%;
  }
  div.form-child-left,
  div.form-child-right {
    flex: 100%;
  }
  div.modal-content {
    margin: 100px auto;
    width: 450px;
    position: relative;
  }
}
```

Feel free to make more changes according to your viewing experience we just need to make the form proper and readable

**Comment by theBeginner86** [comment]
I made the requested changes, and also added a separate media query to handle uncertainty in the responsiveness. Let me know how it looks now.

File: src/components/style.css [hunk 5]
```diff
@@ -358,6 +358,93 @@ ul {
...
+  select.custom-select,
+  input.form-control {
+    width: 80%;
+  }
+  div.form-child-left,
+  div.form-child-right {
+    flex: 50%;
+  }
+  div.modal-content {
+    margin-top: 100px;
```
Comment (Namanl2001) [inline src/components/style.css:396]:
```suggestion
    margin-top: 300px;
```

The heading is missing in this range. Please make the suggested change.

**Review by Namanl2001 — APPROVED** [review]
LGTM! Thanks for the super quick changes. Thanks
Feel free to pick more issues.

**Comment by theBeginner86** [comment]
> LGTM! Thanks for the super quick changes.
> Feel free to pick more issues.

I am happy to help. 😁

**Pull request merged**